python/tests/pyValidator.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# if a command returns a non zero return value, exit script
+set -e
+
+# get current time
+test_directory=test_`date +%s`
+mkdir $test_directory
+
+# save location of SeedlingOntology (validateAIF requires absolute path)
+pushd ../../src/main/resources/com/ncc/aif/ontologies
+ontology_dir=`pwd`/SeedlingOntology
+popd
+
+# run Examples.py
+# DIR_PATH is the directory where example files are to be written
+DIR_PATH=$test_directory python3 Examples.py
+
+# validate test file directory
+../../target/appassembler/bin/validateAIF --ont $ontology_dir -d $test_directory
+
+# if all files are valid, delete the test file directory
+rm -r $test_directory
```

Comment:
It looks like we are relying on the fact that the validator will throw an exception when validation fails which will force a failure of this script and no ultimately never call this `rm -r` command. Is that correct? I am not closely tracking all the changes @nextcen-dgemoets is currently making to the AIF validator, but I believe he is updating how failures are handled. It may be worth making sure that his changes won't inadvertently break the functionality of this script.

Reply:
I will sync up with him today about this. All other requested changes have been made.

Reply:
Typically, the validator *won't* throw an Exception and terminate abnormally.  The [Validator Return Values](https://github.com/NextCenturyCorporation/AIDA-Interchange-Format/#validator-return-values) section of the AIF README describes the return codes.  If you only want to delete test directories when validation is successful, then look for a return code of zero.

Also note that in [PR#165](https://github.com/NextCenturyCorporation/AIDA-Interchange-Format/pull/165) for [AIDA-711](https://nextcentury.atlassian.net/browse/AIDA-711), I am adding a `-o` switch for sending validation output to a file.  This may or may not help you here, and of course it's not quite in master yet.  Maybe by the end of the day?